src/plugins/part.c

```diff
@@ -25,6 +25,7 @@
 #include <inttypes.h>
 #include <unistd.h>
 #include <sys/file.h>
+#include <fcntl.h>
```

Comment:
Why is this new include needed here?

Reply:
That is required.
```
part.c:232:14: error: implicit declaration of function 'open' is invalid in C99
      [-Werror,-Wimplicit-function-declaration]
    dev_fd = open (disk->dev->path, O_RDONLY|O_CLOEXEC);
             ^
part.c:232:37: error: use of undeclared identifier 'O_RDONLY'
    dev_fd = open (disk->dev->path, O_RDONLY|O_CLOEXEC);
                                    ^
part.c:232:46: error: use of undeclared identifier 'O_CLOEXEC'
    dev_fd = open (disk->dev->path, O_RDONLY|O_CLOEXEC);
                                             ^
```

Reply:
Interesting, this doesn't happen for me. But you are right, `open` is defined in `fcntl.h`. Probably something else includes it on Linux.